react/features/overlay/middleware.ts

```diff
@@ -8,7 +8,7 @@ import {
 } from '../base/lib-jitsi-meet/functions.any';
 import StateListenerRegistry from '../base/redux/StateListenerRegistry';
 
-import { openPageReloadDialog } from './actions';
+import { openPageReloadDialog } from './actions.native';
```

Comment:
Isn't this mixing native into the web?

Reply:
This middleware is now part of middlewares.native. Before it was part of middlewares.any.

Reply:
But then shouldn't this be middleware.native then?

Reply:
I can add an .native extension to the file to be more specific and avoid confusion, yes.

Reply:
👍